### PR TITLE
Remove setting for close all tabs pop up

### LINKED ISSUE #8757: Settings UI should hide all warning settings
All warning-related settings should be removed from the Settings UI (at least for now). This includes:
- Appearance > Show close all tabs popup

The introduction of #8324 will create a much cleaner story for warnings in Windows Terminal. After #8324, we should probably introduce a button to "reset all warnings" or something like that, but the game plan for now is...
- remove all warning-settings from the Settings UI
- implement #8324

### PR DESCRIPTION
**Pull request by PankajBhojwani**
Remove the setting for enabling/disabling the dialog that
shows up to confirm the closing of all tabs in the SUI

Closes #8757

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml [hunk 1]
```diff
@@ -64,13 +64,6 @@ the MIT License. See LICENSE in the project root for license information. -->
                                    Style="{StaticResource RadioButtonsSettingStyle}"/>
             </ContentPresenter>
 
-            <!--Confirm Close All Tabs-->
-            <ContentPresenter Style="{StaticResource SettingContainerStyle}">
-                <CheckBox x:Uid="Globals_ConfirmCloseAllTabs"
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml:69]: Please update the .resw file too.

Comment (PankajBhojwani) [reply src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml:69]: Thanks for catching that! Done

**Review by carlos-zamora — APPROVED** [review]

**Review by zadjii-msft — APPROVED** [review]

**Review by DHowett — APPROVED** [review]

**Comment by jamesqo** [comment]
Is there a particular reason why this feature was removed? I prefer not to deal with the dialog every time I want to close all tabs, and I'm sure a lot of other users do too.

**Comment by DHowett** [comment]
The setting was removed _from the UI_ because we’re replacing it with checkboxes that allow you to tell us “don’t ask again”. You can still disable them in the JSON file as always.

**Pull request merged**